Color/Colors.php

```diff
@@ -167,150 +167,162 @@ class Colors
      *
      * @return array
      */
-    public static function getAll()
+    public static function getAll($isInverted = true)
```

Comment:
@verheyenkoen @core23 I have an even better (I hope) idea. How about implementing a type that creates `<input type="color" />`, and deprecate this one? Not sure why sf didn't do it itself already.